layershellev/examples/simplelayer.rs

```diff
@@ -23,6 +23,15 @@ fn main() {
             // NOTE: this will send when init, you can request bind extra object from here
             LayerEvent::InitRequest => ReturnData::RequestBind,
             LayerEvent::BindProvide(globals, qh) => {
+                // Example: pass through input events, useful for overlays
+                // use ::wayland_client::protocol::wl_compositor;
+                // if let Some(compositor) = globals.bind::<wl_compositor::WlCompositor, _, _>(&qh, 1..=5, ()).ok() {
+                //     if let Some(state) = ev.get_unit_with_id(index.unwrap()) {
```

Comment:
Wl_compositer is already in the `running`, why not add another returnvalue for it?